src/Microsoft.Framework.Configuration.File/FileConfigurationbuilderExtensions.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Framework.Configuration
+{
+    public static class FileConfigurationBuilderExtensions
```

Comment:
Given my (lame) proposed package name, the matching type name would be `FileHelpersConfigurationBuilderExtensions`.